Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Standard Dockerfile for GOV.UK services. Random service-specific stuff should *not*
```

Comment:
Perhaps we can call this file something like `govuk-base.dockerfile` to differentiate it from the other `Dockerfiles`?

Reply:
I've gone with 'Dockerfile.govuk-base', which seems to be the convention for variants of Dockerfiles.